packages/cli/src/commands/test.ts

```diff
@@ -108,13 +109,16 @@ export default class Test extends Command {
       }, <Record<string, Runtime>> {}),
       checklyConfigConstructs,
     })
-    const { checks: checksMap, groups: groupsMap } = project.data
-    const checks = Object.entries(checksMap)
-      .filter(([_, check]) => {
-        return filterByFileNamePattern(filePatterns, check.scriptPath) ||
-          filterByFileNamePattern(filePatterns, check.__checkFilePath)
+    const checks = Object.entries(project.checks)
+      .filter(([, check]) => {
+        if (check instanceof BrowserCheck) {
```

Comment:
`check` now has the type `Check` rather than `any`. I changed this since TS had an error about `scriptPath` not being on `Check`. Another solution would be to just use `any` here.